workos/resources/events.py

```diff
@@ -58,6 +137,90 @@ class EventModel(WorkOSModel, Generic[EventTypeDiscriminator, EventPayload]):
     created_at: str
 
 
+class AuthenticationEmailVerificationSucceededEvent(
+    EventModel[
+        Literal["authentication.email_verification_succeeded"],
+        AuthenticationEmailVerificationSucceededPayload,
+    ]
+):
+    event: Literal["authentication.email_verification_succeeded"]
```

Comment:
This boilerplate isn't large, but if you have as many events as we do, it adds up.